cvmfs/catalog_mgr_impl.h

```diff
@@ -550,8 +550,9 @@ CatalogT* AbstractCatalogManager<CatalogT>::FindCatalog(
  * @return true if catalog is already present, false otherwise
  */
 template <class CatalogT>
-bool AbstractCatalogManager<CatalogT>::IsAttached(const PathString &root_path,
-                                        CatalogT **attached_catalog) const
+bool AbstractCatalogManager<CatalogT>::IsAttached(
```

Comment:
Please don't change the style of existing code unless it violates the linter or you are actually modifying things.  Such changes easily take the focus away from the actual purpose of the patch.